build.sbt

```diff
@@ -75,3 +70,15 @@ lazy val checkerPlugin = (projectMatrix in file("checker-plugin"))
   .settings(libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test))
   .dependsOn(checkerLibrary)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
+
+lazy val schemaDumpPlugin = (projectMatrix in file("sbt-dumpschema"))
```

Comment:
Pls include the word `event`... maybe `sbt-dump-akka-event-schemas`?

Reply:
(here and in other places ofc)

Reply:
I'm not sure about this `event` wording. Indeed, the main concern may be events, but the plugin itself doesn't distinguish between events, messages or any other stereotype.

Reply:
Hmmm the problem is that we most likely don't want to include messages and persistent state in the dump... since that's gonna increase the size of the dump, thus diverting attention from the changes in events that we really care about :/

Reply:
I know that _as for now_ we don't distinguish b/w messages/events/state... but I think this is still doable by recognizing where (as in, as what type param) the given class has been used 🤔

Reply:
Could you provide information on how to distinguish events from other classes in code? It may be possible to implement it without loss of generality.

Reply:
Hmm that's the same problem as in the task https://github.com/VirtusLab/akka-safer-serializer/issues/34

The only way to distinguish state/messages/events is to see how they're used in type params of classes like `ReplyEffect`/`Behavior` etc... at least I'm not aware of any other means :/